src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1014,43 +1014,38 @@ public override BoundNode VisitConvertedTupleLiteral(BoundConvertedTupleLiteral
 
         private BoundNode VisitTupleExpression(BoundTupleExpression node)
         {
-            VisitArguments(node.Arguments, default(ImmutableArray<RefKind>), null, default(ImmutableArray<int>), false);
+            VisitArguments(node.Arguments, default(ImmutableArray<RefKind>), null);
```

Comment:
Nit: Naming the `null` with `method:` would help. Here and a few other locations.

Reply:
Actually, a number of the changes in `PreciseAbstractFlowPass`, such as the changes to `VisitArguments`, are reverting the changes in the branch back to the state of `upstream/master`. To minimize the diffs in the branch, I'd like to avoid naming the args here.